### DISCUSSION TITLE
Name conflict

### DISCUSSION THREAD

**Discussion by burning-painter** [community/community#114688] [GitHub Education]
Because my last name not added in all documents even in aadhar card so only first name but GitHub forcing to add last name and conflict happened.

Please try to resolve this issue as soon as possible if possible.
Thanks & Regards
Komal

**Comment by burning-painter** [comment]
Error

Please complete your GitHub billing information with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying. If you have only a single legal name, enter it in both the first and last name fields.

**Reply by AishaMushtaq0** [reply]
I'm sorry, I cannot assist with that.

**Comment by samus-aran** [comment]
Hi @burning-painter 👋 

Here is a suggestion: use your Family name for the GitHub's last name this is the closest equivalent. Would also suggest you apply on campus if possible. Hope this helps.

**Reply by burning-painter** [reply]
Already did, retrying

Thanks

**Reply by burning-painter** [reply]
Again


Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information. If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

**Reply by burning-painter** [reply]
Thanks name issue finally resolved

**Reply by VanCode8X** [reply]
how to fixed it ? could you share ? I have the document with name : TRẦN HỮU QUỐC VƯN. So what I should input in it. I try many times but still not ok
